### PR TITLE
Development v1

### PR DESCRIPTION
**Pull request by Andretalik**
Final commit for presentation

### CODE REVIEW

File: Dojo_class.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import random
+from Room_Class import *
+from Person_Class import *
+
+
+class Dojo(object):
+    def __init__(self):
+        self.all_rooms = []
```
Comment (alex-magana) [inline Dojo_class.py]: The all_rooms variable can store room attributes. This should eliminate the need for all_offices and all_living_spaces

File: Dojo_class.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        self.livingspaces = []
+        self.offices = []
+        self.all_people = []
```
Comment (alex-magana) [inline Dojo_class.py]: Apply the concept indicated above to all_people, fellows and staff

File: Dojo_class.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        self.available_livingspaces = []
+        self.unallocated_offices = []
+        self.unallocated_livingspaces = []
+
+    def create_room(self, room_type, rooms_to_make):
+        if len(rooms_to_make) != 0:
+            for room_name in rooms_to_make:
+                if room_type == "office":
+                    office_instance = Office("office", room_name)           # to ensure the instance being worked on
+                    self.all_rooms.append(office_instance)                  # during the function call is singular
```
Comment (alex-magana) [inline Dojo_class.py]: Consider use of a function that does assignment of rooms. Principle of single responsibility.

File: Dojo_class.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                    self.offices.append(office_instance)
+                    self.available_offices.append(office_instance)
+                    print("An {} called {} has been created".format(office_instance.room_type, office_instance.room_name))
+                elif room_type == "livingspace":
+                    livingspace_instance = LivingSpace("livingspace", room_name)
+                    self.all_rooms.append(livingspace_instance)
+                    self.livingspaces.append(livingspace_instance)
+                    self.available_livingspaces.append(livingspace_instance)
+                    print("A {} called {} has been created".format(livingspace_instance.room_type, livingspace_instance.room_name))
```
Comment (alex-magana) [inline Dojo_class.py]: Limit length of lines to 72 characters.
https://www.python.org/dev/peps/pep-0008/

**Review by kimobrian — COMMENTED** [review]
+1 for a .gitignore
A part from redundancy and code optimization , you really understand your code base. 
+1 for branching your repo.
-Learn about setup and teardown in testing.
-Learn more on data structures(lists, dictionaries etc) esp list comprehensions and control structures(if, while etc)
-Learn on pep8 style of writing code.

**Pull request closed without merging**